EDDI/ChangeLog.md

```diff
@@ -1,4 +1,14 @@
-# CHANGE LOG
+﻿# CHANGE LOG
+
+### 2.4.3
+  * Core
+    * Clarified & modified Help.md to clearly state which functions use SSML tags (like phonetic speech).
```

Comment:
This is a user-facing change log so let's not overwhelm the user with too much detail.